reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -2,11 +2,17 @@
   overflow-y: auto;
   max-height: 600px;
   height: 100%;
-  background-color: #ffffff;
+  background-color: white;
 }
 
-.heatmap table {
-  overflow-y: auto;
+.darkmode .heatmap {
+  background-color: rgb(64, 64, 64);
+  box-shadow: none;
+  border-radius: 10px;
```

Comment:
Why is the border-radius light/darkmode specific?

Reply:
✅